out.py

```diff
@@ -15,18 +15,19 @@ class _CursorInfo(ctypes.Structure):
 
 
 def error(message: str, context: Any = __UNIT) -> None:
-    print(message, file=sys.stderr)
+    println(danger(message), file=sys.stderr)
     if context is not __UNIT:
         print(context, file=sys.stderr)
 
 
 RESET = "\033[0m"
-PRIMARY = "\033[95m"
+PRIMARY = "\033[92m"
 SECONDARY = "\033[94m"
+DANGER = "\033[91m"
 
 
-def println(*args, end: str = "\n") -> None:
-    sys.stdout.write(" ".join(map(str, args)) + end)
+def println(*args, end: str = "\n", file: Any = sys.stdout) -> None:
```

Comment:
The type of the filestream is `TextIO`

Reply:
by Any I meant the possibility to also pass  string for exact filenames, i.e: "logs.txt".

Reply:
That won't work, that will throw an error. You cannot pass a string into the `file` kwarg in the print function. If you look at the definition of `print` you will see it also uses `TextIO` as the type for the file kwarg.

Reply:
Also, we are building a CLI application, that means that logging and printing to the screen are two different things for out. The out module is for printing things the user will see. We can use the built it python `logging` module for debug or application logs